### ISSUE TITLE
ironhide on ASUS U31SD doesn't work.

### ISSUE THREAD

**Issue by nickkossolapov** [MrMEEE/ironhide#99]
Okay, so this is my first computer I've ever used Ubuntu on, so sorry if I'm a total newb. 
I'm trying to install ironhide, and I get 2 errors saying "Initial tests failed.." and "Power configuration tests failed. I'm using a ASUS U31SD, with nVidia GT510M. 
Here is the terminal code: 

nick@nick-U31SD:~$ sudo apt-get install ironhide
[sudo] password for nick: 
Reading package lists... Done
Building dependency tree  
Reading state information... Done
The following NEW packages will be installed:
  ironhide
0 upgraded, 1 newly installed, 0 to remove and 231 not upgraded.
Need to get 0 B/65.3 kB of archives.
After this operation, 1,901 kB of additional disk space will be used.
Selecting previously deselected package ironhide.
(Reading database ... 132810 files and directories currently installed.)
Unpacking ironhide (from .../ironhide_1.0-1~nattyubuntu37_amd64.deb) ...
Processing triggers for ureadahead ...
Setting up ironhide (1.0-1~nattyubuntu37) ...
0
AUTO
_PS0 Disabling nVidia Card Succeded.
_PS0 Enabling nVidia Card Succeded.
- Starting Ironhide X server ironhide                                          Ironhide hasn't been configured for power management, please run ironhide-configuration.
  ... \* The nvidia driver was not loaded and the nouveau module should be deactivated                                                                              \* Try: sudo rmmod nouveau && sudo modprobe nvidia-current                       \* The Ironhide Xserver failed to start. Please check /var/log/Xorg.8.log[fail] 
- Stopping Ironhide X server ironhide                                   [fail] 
- Starting Ironhide X server ironhide                                          _PS0 Enabling nVidia Card Succeded.
  ... \* The nvidia driver was not loaded and the nouveau module should be deactivated                                                                              \* Try: sudo rmmod nouveau && sudo modprobe nvidia-current                       \* The Ironhide Xserver failed to start. Please check /var/log/Xorg.8.log[fail] 
- Stopping Ironhide X server ironhide                                   [fail] 
  System start/stop links for /etc/init.d/ironhide already exist.
  Adding user nick to group ironhide

**Comment by iskeledz** [comment]
Looking at the output, you should probably disable the experimental Nvidia driver (nouveau) and enable the proprietary Nvidia drivers (nvidia-current) in Control Center - Additional Drivers (or whatever it's called in Ubuntu), restart, and then rerun ironhide-configuration. Also, it looks like you're not installing the dependencies of ironhide - I personally added https://launchpad.net/~mj-casalogic/+archive/ironhide/ to my Software Sources and installed ironhide from Synaptic Package Manager which installed all the dependencies as well. I'm a total noob at Linux too so you shouldn't really expect this to work.

**Comment by MrMEEE** [comment]
Follow @iskeledz info :D

**Comment by nickkossolapov** [comment]
Okay, I'm totally lost. When I go to Additional Drivers, the only option is gives me is to enable "NVIDIA accelerated graphics driver," and when I enable that and restard, Unity won't work and I have to use classic mode. And when I added the source, some of the files(I really don't know what to call them :L) that were listed when I updated "failed", When I tried to launch ironhide, the icon appeared in the top right, but I couldn't enter any of the configurations.
I really have no idea what I'm doing :L this is my first time using an OS other than Windows. And also, will in Ubuntu 11.10 some of there issue be solved? Like with the nVidia Optimus and suspend/sleep issues?
